tests/api/test_account.py

```diff
@@ -408,6 +452,180 @@ def test_customer_update(
     assert not data['user']['isActive']
 
 
+def test_logged_customer_update(user_api_client, address):
+    query = """
+    mutation UpdateCustomer($id: ID!, $billing: AddressInput,
```

Comment:
Please extract this query to some sort of a constant and reuse it in each test. It's getting kind long here

Reply:
Also, we are testing the whole flow in `test_customer_update` test, is it really necessary to check it twice? Can't we just update a simple field, like name, to check if it's working properly?